crowdsourcing/serializers/task.py

```diff
@@ -96,6 +97,19 @@ def get_module(self, obj):
     def get_project_name(self, obj):
         return obj.task.module.project.name
 
+    def get_task_with_data_and_results(self, obj):
+        task = TaskSerializer(instance=obj.task, fields=('id', 'task_template')).data
+        template = task['task_template']
+        task_worker_results = TaskWorkerResultSerializer(instance=obj.task_worker_results, many=True,
+                                                            fields=('template_item_id', 'result')).data
+        for task_worker_result in task_worker_results:
+            for item in template['template_items']:
+                if task_worker_result['template_item_id'] == item['id'] and item['role'] == 'input' \
+                and task_worker_result['result'] is not None and item['type'] != 'labeled_input':
+                    item['values'] = task_worker_result['result']
```

Comment:
so dont replace the values and let front end do the work to populate the answer?